draft-packages/tag/KaizenDraft/Tag/Tag.tsx

```diff
@@ -103,6 +118,13 @@ const Tag = (props: TagProps) => {
                 </span>
               )
             }
+            if (variant === "profile") {
```

Comment:
For future reference: You just added on to the pattern as it was, but I really like the `switch(true)...` pattern for these kind of scenarios, rather than doing all these `if(...){ return ...)` statements:
https://medium.com/trabe/using-switch-true-in-javascript-986e8ad8ae4f